frontend/webadmin/modules/frontend/src/main/resources/org/ovirt/engine/ui/frontend/AppErrors.properties

```diff
@@ -887,6 +887,7 @@ HOSTS_LIST=Hosts: ${hostsList}
 HOT_PLUG_CPU_IS_NOT_SUPPORTED=Hot plugging a CPU is not supported for cluster version ${clusterVersion} and architecture ${architecture}.
 HOT_PLUG_CPU_IS_NOT_SUPPORTED_FOR_GUEST_OS=Hot plugging a CPU is not supported for the guest OS ${guestOS} and architecture ${architecture}.
 HOT_PLUG_CPU_CONFLICT=Hot plugging a CPU is conflicted with the VM configuration ${configuration}.
+HOT_PLUG_CPU_IS_NOT_SUPPORTED_DEDICATED=Hot plugging a CPU is not supported with the CPU pinning policy ${policy}.
```

Comment:
it's missing in AppErrors.java, no?

Reply:
yes, and actually the one above as well. added both.